src/harness/reference_models/interference/interference.py

```diff
@@ -198,7 +189,6 @@ def getProtectedChannels(low_freq_hz, high_freq_hz):
 
 def findGrantsInsideNeighborhood(grants, protection_point, entity_type):
   """Finds grants inside protection entity neighborhood.
-
```

Comment:
Again here and below, all empty lines are removed, although we always keep one empty line before the "Args" section for readability

Reply:
Added   the empty lines